### PR TITLE
Add license

### PR DESCRIPTION
**Pull request by jonathonherbert**
## What does this change?

Adds an Apache 2.0 license to the project, correcting the oversight mentioned in #129.

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (paperboyo) [inline LICENSE.md:190]: Yyyy… I’m no lawyer… but…

Comment (jonathonherbert) [reply LICENSE.md:190]: yyyy – yes to copyright!
(heh – e.g. https://github.com/scanamo/scanamo/blob/master/LICENCE)

File: NOTICE.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+   Typerighter
+   Copyright 2011-2020 Guardian News & Media Ltd
```
Comment (akash1810) [inline NOTICE.md:2]:
```suggestion
   Copyright 2011-2021 Guardian News & Media Ltd
```

File: LICENSE.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright 2020 Guardian News & Media Ltd
```
Comment (akash1810) [inline LICENSE.md:190]:
```suggestion
   Copyright 2021 Guardian News & Media Ltd
```

**Review by akash1810 — APPROVED** [review]

**Comment by jonathonherbert** [comment]
Happy new year? 🙃

**Pull request merged**